Manifests/ManagedPreferencesApple/com.apple.MCXAirPort.plist

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://apple.com/macos</string>
+	<key>pfm_description</key>
+	<string>AirPort settings</string>
+	<key>pfm_documentation_url</key>
+	<string>https://developer.apple.com</string>
```

Comment:
@walkintom can you share or update the direct documentation links on the MCXRedirector & MCXAirPort manifests?  If others are going to contribute to these we'll need that info

Reply:
@apizz I just had a quick search, and I can only find a reference to the following:

**MCXRedirector**

https://github.com/jamf/Jamf-Nation-Managed-Preference-Manifests/blob/master/com.apple.MCXRedirector.manifest
https://mosen.github.io/profiledocs/payloads/mac/mcxredirector.html#payloadtype-com-apple-mcxredirector

**MCXAirPort**

https://github.com/jamf/Jamf-Nation-Managed-Preference-Manifests/blob/master/com.apple.MCXAirPort.manifest

I've tested the profile generated by the MCXAirPort and it works, but haven't tested MCXRedirector. I assumed that folder redirection was deprecated?

Reply:
Must admit I completely forgot I added this - MCXRedirector _was_ apparently scheduled to be deprecated, but currently still works - sadly I can't find any documentation for it.